### ISSUE TITLE
quarto-preview does not refresh

### ISSUE THREAD

**Issue by shirdekel** [quarto-dev/quarto-emacs#5]
Running `M-x quarto-preview` on a `.qmd` file previews as expected, but does not refresh when the document is changed and saved. The expected behaviour can be seen when running `quarto preview file.qmd` in a shell.

**Comment by jmburgos** [comment]
Perhaps mention this in the main site?  According to the current description, "Start a quarto preview server that watches quarto content for changes and automatically refreshes it".

**Comment by cscheid** [comment]
Good catch, that's stale documentation.
EDIT: fixed now.

**Comment by henningninneh** [comment]
For me that's `M-x markdown-preview` or `M-x markdown-export`. I was also very confused about this issue, until I found this thread.

**Comment by dalanicolai** [comment]
EDIT

Haha, just noticed this was already mentioned in the original issue. Anyway, might be handy to leave this message here...

END EDIT

I would say a better workaround is to start the `quarto preview ...` manually from a terminal. Then the preview update works as expected...

But I do not understand this design decision. Why does it have to match the behavior of other markdown modes (b.t.w. also it does not match the behavior of [markdown-soma](https://github.com/jasonm23/markdown-soma)?

Quarto looks otherwise great b.t.w.! Thanks for this nice software...

**Comment by henningninneh** [comment]
I agree that this behavior should at least be easily customizable. And, of course, I agree with the praise. :)

**Comment by cscheid** [comment]
@henningninneh emacs is no longer my day-to-day editor and we have a ton of other things to work on, so unfortunately this is relatively low-priority right now. But we appreciate pull requests!
